src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/CreateTaskTest.java

```diff
@@ -22,23 +28,57 @@
 import static uk.gov.hmcts.reform.waworkflowapi.api.CreateTaskRequestCreator.requestRespondentEvidenceTaskRequest;
 import static uk.gov.hmcts.reform.waworkflowapi.api.CreateTaskRequestCreator.unmappedCreateTaskRequest;
 import static uk.gov.hmcts.reform.waworkflowapi.api.TransitionBuilder.aTransition;
+import static uk.gov.hmcts.reform.waworkflowapi.config.ServiceTokenGeneratorConfiguration.SERVICE_AUTHORIZATION;
 
-@RunWith(SpringIntegrationSerenityRunner.class)
-public class CreateTaskTest {
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("functional")
+public class CreateTaskTest extends SpringBootFunctionalBaseTest {
+
+
+    @Value("${targets.instance}")
+    private String testUrl;
+
+    @Value("${targets.camunda}")
+    private String camundaUrl;
```

Comment:
+1 this way to inject the test Url and the camunda URL is cleaner than the other way :)

Reply:
Is Serenity running being used anymore for these tests? This is a requirements from test. When I used the serenity running the @Value stopped working which is why it was the other way to load these URLs.

Reply:
I added the configuration as part of the base test so we can extend from SpringBootFunctionalBaseTest.
just needed to define an active profile  and annotate with @SpringBootTest